app.json

```diff
@@ -77,4 +80,4 @@
       "url": "heroku/ruby"
     }
   ]
-}
+}
```

Comment:
ugh. yeah my editor seems to remove that railing line in json files for some reason when I save. Since I need to fix the json anyway for the above catch, I'll see if I can get this resolved as well.